tests/conftest.py

```diff
@@ -42,6 +42,11 @@ def elevators_sam_learning(elevators_domain: Domain) -> SAMLearner:
     return SAMLearner(elevators_domain)
 
 
+@fixture()
+def elevators_sam_learning_ignore(elevators_domain: Domain) -> SAMLearner:
```

Comment:
elevators...ignore_negative_preconditions --- make it clear for others who might read the code.